rootfs/ingest/python/qserv/jsonparser.py

```diff
@@ -184,9 +210,10 @@ def get_regular_table_locations(responseJson: dict) -> List[Tuple[str, int]]:
     """
     locations = []
     for entry in responseJson["locations"]:
-        host = entry["http_host"]
+        fqdns = entry["http_host_name"]
         port = entry["http_port"]
-        locations.append((host, port))
+        fqdn = get_fqdn(fqdns, port)
```

Comment:
Do you want to check for empty `fqdn` here?

Reply:
:+1: